src/frontends/paddle/src/frontend.cpp

```diff
@@ -135,22 +135,32 @@ bool normalize_framework_node(const std::shared_ptr<FrameworkNode>& node,
     return true;
 }
 
-std::istream* variant_to_stream_ptr(const ov::Any& variant, std::ifstream& ext_stream) {
+OPENVINO_SUPPRESS_DEPRECATED_START
+std::istream* variant_to_stream_ptr(const ov::Any& variant, std::fstream& fs, std::stringstream& ss) {
```

Comment:
Can we use fstream to handle case of AlignedBuffer? Any particular reason to use stringstream?

Reply:
Maybe not. @ceciliapeng2011 
- fstream should be open before use but not file can be open.
- The return type is `istream` so we should return a derived type pointer. AFAIK, stringstream as a derived type can be filled with an internal buffer.